lib/hanami/cli_base.rb

```diff
@@ -0,0 +1,30 @@
+require 'thor'
+
+module Hanami
+  class CliBase < Thor
```

Comment:
@davydovanton I think composition is better for roles, while inheritance for behavior specialization.

In this case, all the subclasses of `CliBase` exist to share the code (which is a role in this case) and they don't specialize base class behavior. So composition (via module inclusion) is preferrable here. 😄